openfecwebapp/templates/layouts/main.html

```diff
@@ -16,6 +16,7 @@
   {% endif %}
   {% if api_location %}
   <script>
+    ANALYTICS = {{ use_analytics|json }};
```

Comment:
One quick question here - why is the `|json` filter (I'm assuming filter since this is in the template) needed here?  Is this accounting for safe output?